main/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+module.exports.SERIALPORT_CHANNEL = {
+    OPEN: 'serialport:open',
+    CLOSE: 'serialport:close',
+    WRITE: 'serialport:write',
+    UPDATE: 'serialport:update',
+    SET: 'serialport:set',
+
+    ON_CLOSED: 'serialport:on-close',
+    ON_DATA: 'serialport:on-data',
+    ON_UPDATE: 'serialport:on-update',
+    ON_SET: 'serialport:on-set',
+    ON_CHANGED: 'serialport:on-changed',
+    ON_WRITE: 'serialport:on-write',
+
+    IS_OPEN: 'serialport:is-open',
+};
```

Comment:
```suggestion
module.exports.SERIALPORT_CHANNEL = {
    OPEN: 'serialport:open',
    CLOSE: 'serialport:close',
    WRITE: 'serialport:write',
    UPDATE: 'serialport:update',
    SET: 'serialport:set',

    ON_CLOSED: 'serialport:on-close',
    ON_DATA: 'serialport:on-data',
    ON_UPDATE: 'serialport:on-update',
    ON_SET: 'serialport:on-set',
    ON_CHANGED: 'serialport:on-changed',
    ON_WRITE: 'serialport:on-write',

    IS_OPEN: 'serialport:is-open',
};

export { SERIALPORT_CHANNEL } from './serialport';
```

Would like it to be reverted to this, but for some reason it breaks the renderer build process in Launcher. Does anyone know?

Reply:
I've reverted to use ES6 module import/export, and now I face an issue when trying to run Launcher (`npm run app`).

```bash
> nrfconnect@3.12.1 app /<path_to_project>/pc-nrfconnect-launcher
> cross-env NODE_OPTIONS=--enable-source-maps electron .

App threw an error during load
/<path_to_project>/pc-nrfconnect-launcher/node_modules/pc-nrfconnect-shared/main/index.ts:7
export const SERIALPORT_CHANNEL = {
^^^^^^

SyntaxError: Unexpected token 'export'
    at wrapSafe (internal/modules/cjs/loader.js:986:16)
    at Module._compile (internal/modules/cjs/loader.js:1034:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1108:10)
    at Module.load (internal/modules/cjs/loader.js:935:32)
    at Module._load (internal/modules/cjs/loader.js:770:16)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/<path_to_project>/pc-nrfconnect-launcher/dist/main.js:2013:19)
        -> /<path_to_project>/pc-nrfconnect-launcher/src/main/serialport.ts:9:36
    at Module._compile (internal/modules/cjs/loader.js:1078:30)
A JavaScript error occurred in the main process
Uncaught Exception:
/<path_to_project>/pc-nrfconnect-launcher/node_modules/pc-nrfconnect-shared/main/index.ts:7
export const SERIALPORT_CHANNEL = {
^^^^^^

SyntaxError: Unexpected token 'export'
    at wrapSafe (internal/modules/cjs/loader.js:986:16)
    at Module._compile (internal/modules/cjs/loader.js:1034:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1108:10)
    at Module.load (internal/modules/cjs/loader.js:935:32)
    at Module._load (internal/modules/cjs/loader.js:770:16)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/<path_to_project>/pc-nrfconnect-launcher/dist/main.js:2013:19)
        -> /<path_to_project>/pc-nrfconnect-launcher/src/main/serialport.ts:9:36
    at Module._compile (internal/modules/cjs/loader.js:1078:30)
^C/<path_to_project>/pc-nrfconnect-launcher/node_modules/electron/dist/electron exited with signal SIGINT

```